eng/referenceFromRuntime.targets

```diff
@@ -45,7 +45,7 @@
   <Target Name="GetFilesFromCoreCLR" 
           Condition="'$(RuntimeFlavor)' != 'Mono'"
           Returns="@(CoreCLRFiles)"
-          DependsOnTargets="ResolveCoreCLRFilesFromLocalBuild" />
+          DependsOnTargets="ResolveRuntimeFilesFromLocalBuild" />
```

Comment:
Why did you change this target but not the `GetFilesFromMono` below? Those targets should mirror each other in this respect.

Reply:
I didn't want to change too much until after I got something working.  Now that I look again, I don't think it's a big deal to alter.